webpack.config.js

```diff
@@ -131,7 +131,8 @@ module.exports = {
   resolve: {
     alias: {},
     modules: [
-      path.join(__dirname, 'node_modules')
+      path.join(__dirname, 'node_modules'),
+      path.join(__dirname, 'node_modules/@parity/ui/node_modules')
```

Comment:
Any specific reason this is needed?

Reply:
yup, webpack couldnt find the dependencies of @parity/ui... wasn't sure how to get this working, I realise it's more than a bit hacky :P